.github/CODEOWNERS

```diff
@@ -6,7 +6,7 @@
 # used for the 'team' designator within Kibana Stats
 
 # Alerting commmunal ownership
-/x-pack/plugins/rule_registry/ @elastic/security-detections-response @elastic/actionable-observability @elastic/response-ops
+/x-pack/plugins/rule_registry/ @elastic/response-ops @elastic/actionable-observability @elastic/security-detections-response-alerts @elastic/security-solution-platform
```

Comment:
@dhurley14 @peluja1012 Does the Security Platform team still own any functionality in the Rule Registry?

@marshallmain Would you prefer to stay a code owner here?

Reply:
I would prefer not to be a codeowner, we keep getting pinged for PRs that don't need our review